src/test/java/org/broadinstitute/hellbender/tools/WalkerOutputIntegrationTest.java

```diff
@@ -0,0 +1,40 @@
+package org.broadinstitute.hellbender.tools;
+
+import htsjdk.samtools.SamReaderFactory;
+import org.broadinstitute.hellbender.CommandLineProgramTest;
+import org.broadinstitute.hellbender.utils.test.BaseTest;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.io.File;
+import java.io.IOException;
+
+public class WalkerOutputIntegrationTest extends CommandLineProgramTest {
```

Comment:
And move `print_reads_withPG.bam` to the engine test data dir as well, and rename it to remove mention of `PrintReads`, since this is not a `PrintReads` test.